stable/fairwinds-insights/templates/_env.yaml

```diff
@@ -33,6 +33,12 @@ env:
 - name: INSTALLATION_CODE
   value: {{ . | quote }}
 {{- end }}
+{{- with .Values.installationCodeSecret }}
+- name: INSTALLATION_CODE
+  secretKeyRef:
+    name: {{ printf "%s-%s" .Release.Name .Values.installationCodeSecret }}
```

Comment:
mmh is there a way you are generating this secret? if is `.Values.installationCodeSecret` that the user supplies why we have release name here?

Reply:
🤦 copypasta error. Good eyes!